internal/provisioner/fluentbit.go

```diff
@@ -80,12 +79,38 @@ func (f *fluentbit) NewHelmDeployment(logger log.FieldLogger) *helmDeployment {
 	if err != nil {
 		logger.WithError(err).Error("unable to lookup private zone name")
 	}
+	domainNameSplitted := strings.Split(privateDomainName, ".")
+	sumoLogic := `[OUTPUT]
+	Name  forward
+	Match  *
+	Host  example.placeholder.mattermost.com
+	Port  %s
+	tls  On
+	tls.verify  Off
+	#tls.ca_file  /etc/certs/ca-chain.pem`
+
+	env := domainNameSplitted[1]
+	var sumoLogicConf string
+	if env == "prod" {
+		sumoLogicConf = fmt.Sprintf(sumoLogic, "24222")
+	} else if env == "staging" {
+		sumoLogicConf = fmt.Sprintf(sumoLogic, "24011")
+	} else if env == "test" {
+		sumoLogicConf = fmt.Sprintf(sumoLogic, "24022")
+	}
```

Comment:
If we don't get a match should we do something or error, or is this okay?

Reply:
we want to be empty as if it is a new environment or dev we don't want to send audit logs on a wrong port, I had an else configuration like that one:
```
else {
		sumoLogicConf = fmt.Sprintf(" ")
	}
```
but I thought it was pointless

Reply:
Okay, sounds good! Yeah, if it is okay if the value is blank then no need to do anything. It's default value will be `""` which is perfect.